apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts

```diff
@@ -960,3 +960,46 @@ export function useHideBottomTabBar() {
     return () => navigation.getParent()?.setOptions({tabBarStyle: true, tabBarVisible: undefined})
   }, [navigation])
 }
+
+const supportedTypes = ['image/png', 'image/jpeg', 'image/gif', 'image/webp', 'image/svg+xml', 'image/tiff']
```

Comment:
Yes! That's the backend list. This is here to not do unnecessary calls to the backend when we know it won't support the image.
If we ask for an unsupported one we get an error. If `enabled` is false (because it's not in this list) react query returns initialData, and since none is provided, it's undefined. 

Both for errors and undefined data, preview shows the placeholder image.